server/routes/handlers/helpers/viewWithRecall.ts

```diff
@@ -7,23 +7,15 @@ import { referenceData } from '../../../referenceData'
 import { dossierDueDateString, recallAssessmentDueText } from './dates/format'
 import { enableDeleteDocuments } from '../documents/upload/helpers'
 import { decorateDocs } from '../documents/download/helpers/decorateDocs'
-import { getPerson } from './personCache'
 import logger from '../../../../logger'
 
-const requiresPerson = (viewName: ViewName) =>
-  ['assessRecall', 'dossierRecallInformation', 'viewFullRecall', 'recallCheckAnswers'].includes(viewName)
-
-export const viewWithRecallAndPerson =
+export const viewWithRecall =
   (viewName: ViewName) =>
   async (req: Request, res: Response): Promise<void> => {
     const { nomsNumber, recallId } = req.params
-    const [recallResult, personResult] = await Promise.allSettled([
-      getRecall(recallId, res.locals.user.token),
-      requiresPerson(viewName) ? getPerson(nomsNumber as string, res.locals.user.token) : undefined,
-    ])
-    if (recallResult.status === 'rejected' || personResult.status === 'rejected') {
-      logger.info('Error getting recall or person', {
-        personResult: (personResult as PromiseRejectedResult).reason,
+    const [recallResult] = await Promise.allSettled([getRecall(recallId, res.locals.user.token)])
```

Comment:
only one call now, so can just be:
`const recall = await getRecall()`

Reply:
Can remove the 'rejected' error handling?

Reply:
yes; a thrown error will end up in the errorHandler.ts module which rendered the same error page anyway